app/models/setting.rb

```diff
@@ -83,4 +88,30 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  def self.violations_count_regulations_link
+    hash = Setting.find_by(code: :violations_count_regulations_link)&.value
+    hash.present? ? JSON.parse(hash).with_indifferent_access[I18n.locale] : nil
+  end
+
+  private
+
+  def validate_value
+    validation_methods_for_code = FORMAT_VALIDATIONS[code]
+    return unless validation_methods_for_code
+
+    validation_methods_for_code.each do |validation_method|
```

Comment:
What is the disadvantage of replacing meta programming with simple method calls? The disadvantage of meta programming is the complexity it introduces. For such simple goal it seems like overkill.

Reply:
When we'll add Karl's and another mine PR on Setting with this methods, all we need to do will just to add validator method names to FORMAT_VALIDATIONS hash.
Yes, right here it might be overkill, but there is two more PRs with same functionality which are fitting seamlessly there, I suppose.